test/aave/CloseToCollateral.test.ts

```diff
@@ -258,23 +252,22 @@ describe('Close AAVEv2 Position to collateral', () => {
     const expectedBalance = userCollateralBalanceBeforeTx.plus(
       currentPosition.collateral.amount.minus(closeStrategy.simulation.swap.fromTokenAmount),
     )
-
-    // Given the nature of stETH, there is 1 WEI remaining within astETH and cannot be withdrawn
-    // thus there is a difference if we compare the two numbers up to the last wei.
-    // Using 16 precision will still give us accurate results.
-    // It's 16 and not 17 because if there is the following case:
-    // 1967693420651624420 -> (to 17) 1.96769342065162442
-    // 1967693420651624419 -> (to 17) 1.96769342065162441
-    // So 16 is a safe bet.
+    const feeWalletChange = feeRecipientBalanceAfterTx.minus(feeRecipientBalanceBeforeTx)
+
+    // The precision of the two comparison amounts has been lowered to 8.
+    // This is because in the time between us getting the current position
+    // And executing the transaction, the collateral accrues interest.
+    // When we withdraw the collateral with the MAX_UINT flag we get a different collateral amount
+    // To the amount we had earlier after querying the position
+    // By lowering the sensitivity of the comparison we can avoid this issue
     expectToBeEqual(
-      amountFromWei(expectedBalance).toFixed(16),
-      amountFromWei(userCollateralBalanceAfterTx).toFixed(16),
+      amountFromWei(expectedBalance).toFixed(9),
```

Comment:
It's lowered to 8 in the comment but you put 9 in the `toFixed`

Reply:
Done

Reply:
Corrected 👍